packages/nodejs/.changesets/stop-minutely-probes-when-stopping-appsignal.md

```diff
@@ -0,0 +1,8 @@
+---
+bump: "patch"
+type: "fix"
+---
+
+The minutely probes are now stopped when `Appsignal.stop()` is called.
+In addition, the probes can be stopped independently of the rest of
+the integration by calling `probes.stop()`.
```

Comment:
Not sure if we want to communicate that people can stop it by calling `probes.stop`. Let's keep it to `Appsignal.stop()` for now and we can always document this way of stopping it in the docs if necessary.

We can say something about the probes no longer sticking around after a AppSignal stops but the process continues (like Jest).

Reply:
Done 🎉